### ISSUE TITLE
Provide a role to inject code in the latex output

### ISSUE THREAD

**Issue by nthiery** [jupyter-book/mystmd#613]
One of the (few) remaining reasons to produce LaTeX/PDF exports is when
fine-tuning of the layout is required for printing.

## Proposal

Provide a `latexonly`  role that inserts its content verbatim in the
produced LaTeX exports, and is ignored by all other exporters.

Example:

     {latexonly}`\pagebreak`
     {latexonly}`\bigskip`
     {latexonly}`\enlargethispage{1cm}`

## A use case

I print my exercise sheets for 250 students. I typically end up tuning
the layout by inserting vertical skips, page breaks, page enlargements,
so that exercises don't get split across pages for readability, while
optimizing page numbers (a one page gain saves one package of paper!).
As I am switching my sheets from latex to MyST, I would like to keep
that ability.

## Fun fact

I am currently  desperate enough to hijack in my latex template  `<s></s>`
to insert page breaks  and `<i></i>` to insert vertical spacing :-)

**Comment by rowanc1** [comment]
Related to #714.

**Comment by nthiery** [comment]
For information, in case this can be useful to others (@alanlujan91?): I cooked myself
a quick and (very) dirty latexonly extension for my urgent needs:

https://gitlab.dsi.universite-paris-saclay.fr/Info111/Info111/-/blob/master/extensions/latexonly.mjs?ref_type=heads

Beware:
- It should really be implemented on the transformers side to behave differently for latex and other exports. For now, I am using a dedicated `myst.yml` for my latex export which loads this extension.
- The latex exporter quotes latex commands embedded in text nodes. I hacked around this by building math nodes and escaping out of math mode with additional `$`.
